core/src/apps/cardano/sign_tx/signer.py

```diff
@@ -61,9 +66,22 @@
 TX_BODY_KEY_COLLATERAL_INPUTS = const(13)
 TX_BODY_KEY_REQUIRED_SIGNERS = const(14)
 TX_BODY_KEY_NETWORK_ID = const(15)
+TX_BODY_KEY_COLLATERAL_RETURN = const(16)
+TX_BODY_KEY_TOTAL_COLLATERAL = const(17)
+TX_BODY_KEY_REFERENCE_INPUTS = const(18)
+
+BABBAGE_OUTPUT_KEY_ADDRESS = const(0)
+BABBAGE_OUTPUT_KEY_AMOUNT = const(1)
+BABBAGE_OUTPUT_KEY_DATUM_OPTION = const(2)
+BABBAGE_OUTPUT_KEY_REFERENCE_SCRIPT = const(3)
+
+DATUM_OPTION_KEY_HASH = const(0)
+DATUM_OPTION_KEY_INLINE = const(1)
 
 POOL_REGISTRATION_CERTIFICATE_ITEMS_COUNT = 10
 
+MAX_CHUNK_SIZE = 256
```

Comment:
1024 should be a safe value that is also used elsewhere (Ethereum data streaming, Bitcoin extra_data)

Reply:
Alright 👍 Updated in https://github.com/trezor/trezor-firmware/pull/2354/commits/2cb42cacb84efd27ae9bbe1b7caa53694dc0d252 and also the test case has been updated in https://github.com/trezor/trezor-firmware/pull/2354/commits/523417280374c4f4f729b2475b45fafc18a09175.